Resources/Textures/Clothing/Shoes/Boots/winterbootssci.rsi/meta.json

```diff
@@ -0,0 +1,26 @@
+{
+  "version": 1,
+  "license": "CC-BY-SA-3.0",
+  "copyright": "The owner of the picture is UbaserB. Edited by Kilk." ,
```

Comment:
Please format all the copyrights like this "Made by UbaserB (NOTE preferably the github name here or another public one) and edited by Kilk (NOTE same) for SS14"